static/javascript/tba_js/eventwizard_apiwrite.js

```diff
@@ -326,6 +326,29 @@ $('#rankings_file').change(function(){
     reader.readAsBinaryString(f);
 });
 
+$('.update-rankings').click(function(){
+    $(this).css('background-color', '#eb9316');
+    $.ajax({
+            url: '10.0.100.5/pit/getdata?random=' + Math.random(),
+            cache: false,
+            timeout: 5000, success: function (data) {
+                rankings = ['Rank', 'Team', 'Avg', 'CP', 'AP', 'RC', 'TP', 'LP', 'Played'];
+                rankData = JSON.parse(data);
+                for(var i=0; i<rankData.length; i++){
+                    // Turn team number -> team key
+                    rankData[i]['Team'] = "frc"+rankData[i]['Team'];
+                    rankings.push(rankData[i]);
+                }
+                makeRequest('/api/trusted/v1/event/' + $('#event_key').val() + '/rankings/update', JSON.stringify(rankings), $(this));
```

Comment:
Is this the right format to make the trusted API call? https://github.com/the-blue-alliance/the-blue-alliance/blob/master/datafeeds/parsers/json/json_rankings_parser.py